src/analyze_repo.py

```diff
@@ -99,4 +108,9 @@ def call_set_commit_stats(h, commit):
     return ret
 
 
-
+def sanitize_filename(path):
```

Comment:
I am afraid it won't be good in all cases. Because `_` or `.` are non-alphanumerical but valid file names.

Reply:
Doing it for just the last char.